schedule/yast/sle/guided_btrfs/guided_btrfs@s390x_kvm.yaml

```diff
@@ -7,6 +7,15 @@ vars:
   HDDSIZEGB: 40
   YUI_REST_API: 1
 schedule:
+  extension_module_selection:
+    - installation/module_registration/register_module_desktop
+  system_role:
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
+  suggested_partitioning:
```

Comment:
let's apply here the same, all under `guided_partitioning`.

Reply:
Modified.